doc/project/checklist.md

```diff
@@ -156,11 +156,11 @@ Review Design Change with SW: Review known "Won't Fix" bugs and "Errata".
 ## V1
 
 For a transition from V0 to V1, the following items are expected be completed.
+Prefix "SIM" is applicable for simulation-based DV approach only, while "FPV" is for FPV approach only.
```

Comment:
does a checked in checklist need to have both SIM_ and FPV_ prefix items, or only one or the other?

Reply:
Ideally, all existing IP checklists would be updated to reflect these changes. If each DV owner does it for their IPs, then it should be a reasonably small effort. I am inclined towards not deleting items from checklists, considering 'NA' is a valid response.